src/core/wb.js

```diff
@@ -255,6 +255,16 @@ var getUrlParts = function( url ) {
 			"#999999"
 		],
 
+		// Get and generate a unique session id
```

Comment:
Because it's needed by https://github.com/wet-boew/GCWeb/pull/1237 which are independent from #7983